<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PRWidget</class>
 <widget class="QWidget" name="PRWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>147</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add new record</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="TextInput" name="edit_name">
        <property name="placeholderText">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_unit">
        <property name="text">
         <string>Unit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="UnitInput" name="edit_unit">
        <property name="placeholderText">
         <string>unit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_expiry">
        <property name="text">
         <string>Expiry</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="DateInput" name="edit_expiry">
        <property name="placeholderText">
         <string>expiry date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="action_addnext">
          <property name="text">
           <string>&amp;Insert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="action_addexit">
          <property name="text">
           <string>Insert and &amp;Exit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="action_clear">
          <property name="text">
           <string>C&amp;lear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="action_cancel">
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextInput</class>
   <extends>QLineEdit</extends>
   <header>TextInput.h</header>
  </customwidget>
  <customwidget>
   <class>DateInput</class>
   <extends>QLineEdit</extends>
   <header>DateInput.h</header>
  </customwidget>
  <customwidget>
   <class>UnitInput</class>
   <extends>QLineEdit</extends>
   <header>UnitInput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edit_name</tabstop>
  <tabstop>edit_unit</tabstop>
  <tabstop>edit_expiry</tabstop>
  <tabstop>action_addnext</tabstop>
  <tabstop>action_addexit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
